src/main/pages/che-7/installation-guide/assembly_setup-che-in-tls-mode-with-self-signed-certificate.adoc

```diff
@@ -0,0 +1,38 @@
+---
+title: Setup Che in TLS mode with self-signed certificate
+keywords:
```

Comment:
Why is keywords list empty?

Reply:
Stupid answer would be "to follow convention" :)
I've added some.